plugins/UsersManager/Model.php

```diff
@@ -187,14 +194,15 @@ public function addUser($userLogin, $passwordTransformed, $email, $alias, $token
     {
         $user = array(
             'login'            => $userLogin,
-            'password'         => $passwordTransformed,
+            'password'         => $this->passwordHelper->hash($passwordTransformed),
```

Comment:
Ideally the password hashing was done outside the model but it's not too important for now (the class should be rather without logic which also makes it easier testable) but in this case it should be fine and we can always change it later